source/integrations/jira.rst

```diff
@@ -1,10 +1,17 @@
 .. _jira:
 
-JIRA Webhook Plugin 
+JIRA Plugin 
 ================================
 
 Set up a JIRA integration for your Mattermost instance within minutes using a pre-packaged JIRA webhook plugin.
 
+**Usage**
+
+Once enabled, JIRA posts notifications to a Mattermost channel when an issue is created, updated or deleted.
+
+.. image:: ../../source/images/jira_sample.png
```

Comment:
Not sure why the image doesn't render..

Reply:
The image doesn't render because it's uploaded directly in "source", not in "source/images"

Reply:
Oh oops..let me fix that